smartapp-core/src/main/java/com/smartthings/sdk/smartapp/core/Response.java

```diff
@@ -2,6 +2,8 @@
 
 import com.smartthings.sdk.smartapp.core.models.*;
 
+
+@SuppressWarnings({"PMD.UseUtilityClass", "PMD.ClassNamingConventions"})
```

Comment:
If someone were to suggest that I should just rename this class to `ResponseUtil` rather than suppressing these warnings, I would happily do it in a heartbeat. I only didn't because it would be a breaking change. We're still in preview mode though, so...